packages/core/src/lib/nunjucks-extensions/set-external.js

```diff
@@ -18,6 +20,7 @@ const logger = require('../../utils/logger');
 class SetExternalExtension {
   constructor(rootPath, env) {
     this.tags = ['ext'];
+    this.acceptedFileTypes = ['json', 'csv'];
```

Comment:
since its not mutated, how about a constant outside of the class? `const acceptedFileTypes = ...`